server/helpers/utils.js

```diff
@@ -0,0 +1,15 @@
+import slug from 'slug';
+import uuid from 'uuid/v4';
```

Comment:
Use uniqid, it's a smaller library.

Reply:
I removed `uuid` and didn't install `uniqid` because I realized I don't need them in this branch.